Documentation~/Diagnostics.md

```diff
@@ -49,12 +49,14 @@ This is a full list of all builtin settings diagnostics:
 | PAS0027 | Mipmap Stripping                            | Player    | Any                      |
 | PAS0028 | Reuse Collision Callbacks                   | Physics   | Any                      |
 | PAS0029 | Splash Screen                               | Player    | Any                      |
+| PAS0030 | Texture Streaming Enabled                   | Quality   | Any                      |
```

Comment:
To be consistent with the current numbering, both should be using PAS1xxx IDs:

| PAS1006 | Texture Mipmap Streaming
| PAS1007 | LightMap Streaming